components/event/org.wso2.carbon.event.core/src/main/java/org/wso2/carbon/event/core/internal/topic/registry/RegistryTopicManager.java

```diff
@@ -137,20 +167,27 @@ public void addTopic(String topicName) throws EventBrokerException {
         }
     }
 
+    /**
+     * Gets a topic name without the resource path
+     *
+     * @param topic topic name
+     * @return a topic name
+     */
```

Comment:
The method name is not relevant to the method comment. Shouldn't it be getResourcePath() ?